### PR TITLE
[Feature] #670767 - Call Page/Widget Layout

### PR DESCRIPTION
**Pull request by adrianoct42**
This PR aims to implement new colors to the app, which consists in a new file labeled DSDarkColors.

### CODE REVIEW

File: lib/src/themes/colors/ds_dark_colors.theme.dart [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+/// All [Color] constants that are used by this Design System.
+abstract class DSDarkColors {
+  static const Color surface0 = Color(0xFF424242);
+  static const Color surface1 = Color(0xFF292929);
+  static const Color surface3 = Color(0xFF141414);
+  static const Color contentDefault = Color(0xFFF6F6F6);
+
+  static const Color extendBlueDark = Color(0xFF1968F0);
```
Comment (mpamaro) [inline lib/src/themes/colors/ds_dark_colors.theme.dart:10]:
```suggestion
  static const Color extendBlue = Color(0xFF1968F0);
```

File: lib/src/themes/colors/ds_dark_colors.theme.dart [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  static const Color extendGreenDark = Color(0xFF35DE90);
```
Comment (mpamaro) [inline lib/src/themes/colors/ds_dark_colors.theme.dart:11]:
```suggestion
  static const Color extendGreen = Color(0xFF35DE90);
```

File: lib/src/themes/colors/ds_dark_colors.theme.dart [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+/// All [Color] constants that are used by this Design System.
```
Comment (mpamaro) [inline lib/src/themes/colors/ds_dark_colors.theme.dart:3]:
```suggestion
/// All Dark [Color] constants that are used by this Design System.
```

**Review by mpamaro — CHANGES_REQUESTED** [review]

**Review by githubdoandre — APPROVED** [review]

File: lib/src/themes/colors/ds_dark_colors.theme.dart (hunk 3 above)
Comment (adrianoct42) [reply lib/src/themes/colors/ds_dark_colors.theme.dart:3]: Done!

File: lib/src/themes/colors/ds_dark_colors.theme.dart (hunk 1 above)
Comment (adrianoct42) [reply lib/src/themes/colors/ds_dark_colors.theme.dart:10]: Done!

**Review by mpamaro — APPROVED** [review]

**Pull request merged**